tests/add-user.test.js

```diff
@@ -89,4 +89,45 @@ describe("Given Item Store is connected to NATS", () => {
             });
         });
     });
+    describe("When I add a new User with a Validation Error [a field (user_id) is wrong-typed]", () => {
+        let response;
+        let message;
+
+        beforeAll(async () => {
+            response = await natsConnection.request(
+                "item-store.add_user",
+                jsonCodec.encode({
+                    user_id: "azerty",
+                    inventory_address: "KT1_SFFG345FFSFdqsfz"
```

Comment:
this is actually incorrect, inventory_address can't be provided when we create a new user, it's added automatically by the tokenization service when we create an inventory contract for the user